examples/isentropic_example.py

```diff
@@ -143,7 +143,8 @@
 # Make some titles
 ax.set_title('{:.0f} K Isentropic Pressure (hPa), Wind (kt), Relative Humidity (percent)'
              .format(isentlevs[level].m), loc='left')
-add_timestamp(ax, times[0].dt, y=0.02, high_contrast=True)
+add_timestamp(ax, times[0].values.astype('datetime64[ms]').astype('O'),
```

Comment:
I'm not at all happy with this one, but I don't have a better solution. Might need to work with upstream to improve this situation.

Reply:
I've found no more efficient way of converting the time values to a date time format that is much easier to use for plotting purposes. Even though this example is working with a single time, this method will convert a whole array of values in one fell swoop without need to loop through each value individually. I admit it is not pretty, but it gets the job done.

Reply:
Yeah, it's the best we can do right now. See pydata/xarray#3256 for upstream issue.